src/Application/Actions/Send.php

```diff
@@ -72,6 +72,11 @@ protected function action(): Response
             return $this->respond(new ActionPayload(400, null, $error));
         }
 
+        $sender = $this->request->getHeaderLine('x-sending-application');
```

Comment:
I'll make a PR make matchbot start sending this. Ideally all apps would send it, but it's OK if they don't - mailer will just display when its not empty.

Reply:
@bdsl Could we use the standard `User-Agent` for this? I think that Salesforce callouts already set that with a generic Salesforce-y UA string by default; it may well be that we could also override that to be specific about the part of Salesforce we're calling from, if desired.